packages/react-theme-provider/src/makeVariantClasses.ts

```diff
@@ -3,6 +3,9 @@ import { tokensToStyleObject } from './tokensToStyleObject';
 import { Variants, Theme } from '@fluentui/theme';
 import { IStyle } from '@fluentui/merge-styles';
 import { makeClasses } from './makeClasses';
+import { isIE11 } from '@fluentui/utilities';
+import { StyleRenderer } from './styleRenderers/types';
+import { useVariantClassesIE11Override } from './useVariantClassesIE11Override';
 
```

Comment:
You should remove `callOrReturn` in line 15 since you're already placing it in another file.

Reply:
done.